lib/src/model/movie.dart

```diff
@@ -0,0 +1,39 @@
+import 'movies_result.dart';
+
+class Movie {
+  late final int page;
+  late final int totalResults;
+  late final int totalPages;
+  late final List<MoviesResult> results;
+
+  Movie({
+    required this.page,
+    required this.totalResults,
+    required this.totalPages,
+    required this.results,
+  });
+
+  factory Movie.fromJson(Map<String, dynamic> parsedJson) {
+    var tempMovies = parsedJson['results'] as List;
```

Comment:
This variable name is not representative enough.

Reply:
Avoid shortest variable names be more specific but not tell a story.

Reply:
Done